modules/monitoring-disabling-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+[id="disabling-alert-routing-for-user-defined-projects_{context}"]
+= Disabling alert routing for user-defined projects
+
+If you have enabled alert routing for user-defined projects, you can disable it. By doing so, you prevent users with the **alert-routing-edit** role from configuring alert routing for user-defined projects in Alertmanager.
+
+[NOTE]
+====
+Alert routing for user-defined projects is disabled by default. You do not need to disable it if the feature is not already enabled.
+====
+
+.Prerequisites
+
+* You have enabled user-defined monitoring.
```

Comment:
You have enabled user-defined projects monitoring

Reply:
Fixed